pallet-logion-vote/src/lib.rs

```diff
@@ -0,0 +1,112 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use frame_support::codec::{Decode, Encode};
+use frame_support::dispatch::Vec;
+use scale_info::TypeInfo;
+
+pub use pallet::*;
+
+#[cfg(test)]
+mod mock;
+
+#[cfg(test)]
+mod tests;
+
+#[derive(Encode, Decode, Clone, PartialEq, Eq, Debug, TypeInfo)]
+pub struct Vote<LocId, AccountId> {
+    loc_id: LocId,
+    ballots: Vec<Ballot<AccountId>>,
+}
+
+#[derive(Encode, Decode, Clone, PartialEq, Eq, Debug, TypeInfo)]
+pub struct Ballot<AccountId> {
+    voter: AccountId,
+    status: BallotStatus,
+}
+
+#[derive(Encode, Decode, Clone, PartialEq, Eq, Debug, TypeInfo)]
+pub enum BallotStatus {
+    UnVoted,
```

Comment:
To me, un-voted means that you voted before and you "cancelled" your vote. Maybe use "not voted" instead?

Reply:
Agreed !